### ISSUE TITLE
Allow option to set quote character?

### ISSUE THREAD

**Issue by minhhien1996** [taozhi8833998/node-sql-parser#94]
In some database such as PostgreSQL, the quote character is not back stick .  Can you allow that in `opts`? Or we can choose the quote character based on the `opts` `database` ?

**Comment by taozhi8833998** [comment]
ok, I will add it in options, then you could define it by yourself

**Comment by taozhi8833998** [comment]
try `1.6.3` version.

```javascript
const opt = {
  database: 'PostgresQL'  //  the same parser with mysql currently, I would add any special pg grammer if you needed laterla
}
const { Parser } = require('node-sql-parser');
const parser = new Parser()
// opt is optional
const ast = parser.astify('select id from db.abc', opt);
const sql = parser.sqlify(ast, opt); // SELECT "id" FROM "db"."abc"
```
Does the above codes could get ur mind??? 
I will close this issue if it not right, reopen it and show me some cases

**Comment by minhhien1996** [comment]
Thanks. `1.6.3` is exactly what I'm looking for

**Comment by minhhien1996** [comment]
I found out that it is not working on the browser, so I have read your commit for the version `1.6.3` and found out the reason is usage of `process.env`. Can we change that?

**Comment by taozhi8833998** [comment]
ok, I will change it later

**Comment by taozhi8833998** [comment]
sorry for bad implementation and late reply, I have published `1.6.4` version, and fixed that. You could try the new version.
